src/hooks/usePollForGetQuestion.ts

```diff
@@ -34,7 +38,7 @@ export function usePollForGetQuestion() {
         const { status, data } = response;
         if (status === 200) {
           console.log("status 200 on usePollForGetQuestion");
-          if (requestNextQuestion && index && (index - 1) === data.index) {
+          if (requestNextQuestion && index && index - 1 === data.index) {
```

Comment:
```suggestion
          if (requestNextQuestion && index && (index - 1) === data.index) {
```

Reply:
https://assets.change.org/photos/8/jw/ax/QMjwAxeQAfcpoxs-800x450-noPad.jpg?1597557421

Reply:
i actually tried adding this and it looks like prettier doesn't like it so it removes the brackets